scout/demo/643595.config.yaml

```diff
@@ -62,3 +62,5 @@ rank_model_version: '1.20'
 rank_score_threshold: -100
 analysis_date: 2016-10-12 14:00:46
 human_genome_build: 37
+
+track: 'cancer'
```

Comment:
Just for testing purposes, because this sample has MT chromosome alignments to trigger the IGV.js browser. TO BE REVERTED IMMEDIATELY BEFORE MERGING!!!